docs/deployment.md

```diff
@@ -132,6 +133,49 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
         $ now scale https://a-fancier-url.now.sh 1
 
+### GitHub Actions
+
+> **Heads Up!** [GitHub Actions](https://github.com/features/actions) is still in limited beta.
+
+GitHub Actions allows you to trigger workflows based on GitHub events, which makes it a great fit for running Probot Apps. To run a your app on GitHub Actions:
+
+1. Add a `Dockerfile` to your app:
+    ```
+    FROM node:10
+
+    ENV PATH=$PATH:/app/node_modules/.bin
+    WORKDIR /app
+    COPY . .
+    RUN npm install --production
+
+    ENTRYPOINT ["probot", "receive"]
+    CMD ["/app/index.js"]
+    ```
+
+  1. In the repository that you want to run the app, create a `.github/main.workflow` file that defines the action and listens for any events that your app depends on. For example, here is the workflow for @jasonetco's [TODO](https://github.com/jasonetco/todo):
+    ```
+    workflow "Check for TODOs in Pull Requests" {
+      on = "pull_request"
+      resolves = "TODO"
+    }
```

Comment:
This looks a bit broken for some reason? Maybe because of the indent in addition to the triple backticks?

![image](https://user-images.githubusercontent.com/7718702/47037691-6c489200-d134-11e8-8605-045b39c7eefb.png)

Reply:
Thanks. Fixed.